src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -790,6 +788,29 @@ bool CommandRequestImpl::IsResultCodeUnsupported(
          (first.is_unsupported_resource && second.is_unsupported_resource);
 }
 
+std::string GetComponentNameFromInterface(const HmiInterfaces::InterfaceID& interface) {
```

Comment:
@pold500 I have checked locally your code with scripts 027_ATF_P_PerformAudioPassThru_UI_timeout_TTS_success.lua
028_ATF_P_PerformAudioPassThru_TTS_timeout_UI_success.lua
and they don't pass. The error is:
expected: UI component does not respond, actual value: Buttons, BasicCommunication, SDL component does not respond
The same situation with any script for TTS, VR not respond. 
Please fix logic and check locally with scripts for reproducing defect